machine/wnc_x86_64/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,48 @@
+get_serial_num()
+{
+    if [ -z "$serial_num" ] ; then
+        if type get_serial_num_platform | grep -q 'shell function' > /dev/null 2>&1 ; then
+            serial_num=$(get_serial_num_platform)
+        elif type get_serial_num_arch | grep -q 'shell function' > /dev/null 2>&1 ; then
+                serial_num=$(get_serial_num_arch)
+        fi
+    fi
+
+    if [ -n "$serial_num" ] ; then
+        echo $serial_num
+    else
+        echo "Unknown"
+    fi
+}
```

Comment:
Is this function different from the default?  If not then this function definition is not required.